clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -970,6 +970,13 @@ public ClientResponse completed(AbstractResponse response, long timeMs) {
         public ClientResponse disconnected(long timeMs) {
             return new ClientResponse(header, callback, destination, createdTimeMs, timeMs, true, null, null);
         }
+        
+        @Override
+        public String toString() {
+            return "InFlightRequest{" + "header=" + header + ", destination='" + destination + '\'' + ", callback=" + callback
```

Comment:
Hmm, why is there a `\` after `destination`?

Reply:
This was autogenerated. Not totally sure either though, removed the quotes around destnation